js/charts/line.js

```diff
@@ -1492,8 +1505,8 @@ function line() {
             .style("opacity", "1");
 
         var area = plot.selectAll('.area')
-            .style("visibility", function (d, i) {
-                if (_lineType[i].toUpperCase() == "AREA") {
+            .attr("visibility", function (d, i) {
+                if (_lineType[_measure.indexOf(d[i].tag)].toUpperCase() == "AREA") {
```

Comment:
pls declare the AREA as constant in a separate js/json file.
reason : if you keep AREA in many places...if you want to change next time then u will need change in all place

Reply:
okay I will do this